backblazeUpload.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/kothar/go-backblaze.v0"
+)
+
+func uploadFileToBackblaze(filename string, accountID string, applicationKey string, bucketName string) (string, error) {
+	b2, err := backblaze.NewB2(backblaze.Credentials{
+		AccountID:      accountID,
+		ApplicationKey: applicationKey,
+	})
+	if err != nil {
+		return "", err
+	}
+
+	bucket, err := b2.Bucket(bucketName)
+	if err != nil {
+		return "", err
+	}
+
+	path := filename
+	reader, err := os.Open(path)
```

Comment:
Rename `reader` to `file`, which matches the name of the arguments in the backblaze documentation (`UploadFile(name string, meta map[string]string, file io.Reader)`)